From f0a6ef67e1b944e2032ce777994257075f6ba107 Mon Sep 17 00:00:00 2001
From: Antonio Rosales <antonio.rosales@canonical.com>
Date: Mon, 6 Aug 2012 09:21:07 -0500
Subject: [PATCH] Fix compile error on Ubuntu 12.10

This is related to an earlier fix made under commit #
0fa5fc43ca2004546864051a584367a50413f190.  Apparently
a newer compiler finds yet another file that required
our config.h to be included prior to any system header
files.

Signed-off-by: Maynard Johnson <maynardj@us.ibm.com>
---
 libutil++/bfd_support.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libutil++/bfd_support.h b/libutil++/bfd_support.h
index 4f6a369..ab448d1 100644
--- a/libutil++/bfd_support.h
+++ b/libutil++/bfd_support.h
@@ -11,6 +11,7 @@
 #ifndef BFD_SUPPORT_H
 #define BFD_SUPPORT_H

+#include "config.h"
 #include "utility.h"
 #include "op_types.h"
 #include "locate_images.h"
